src/main/java/uk/gov/crowncommercial/dts/scale/cat/config/JaggaerOAuth2Config.java

```diff
@@ -97,9 +103,43 @@ public Request newRequest(URI uri) {
     ClientHttpConnector jettyHttpClientConnector = new JettyClientHttpConnector(client);
 
     return WebClient.builder().clientConnector(jettyHttpClientConnector)
-        .baseUrl(jaggaerAPIConfig.getBaseUrl()).defaultHeader(ACCEPT, APPLICATION_JSON_VALUE)
-        .defaultHeader(ACCEPT_CHARSET, UTF_8.name()).apply(oauth2Client.oauth2Configuration())
-        .build();
+        .filter(buildResponseHeaderFilterFunction()).baseUrl(jaggaerAPIConfig.getBaseUrl())
+        .defaultHeader(ACCEPT, APPLICATION_JSON_VALUE).defaultHeader(ACCEPT_CHARSET, UTF_8.name())
+        .apply(oauth2Client.oauth2Configuration()).build();
+  }
+
+  /**
+   * SCC-517
+   *
+   * <p>
```

Comment:
Very minor - does this need a closing tag?

Reply:
I think it can be omitted - can't find exact link but all the samples leave out the closing `</p>` e.g. https://www.oracle.com/uk/technical-resources/articles/java/javadoc-tool.html